From 31df0a854aa40c4ce8fbf1ca02ccc9b6bff0abfd Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Fri, 6 Jul 2018 15:51:15 +0100
Subject: [PATCH] Send user to our wiki instead of Meson bug system

If a CPU family isn't recognised the first step should be to verify the
mapping. Send the user to a wiki page explaining what to do, instead of
directly to the Meson bug tracker.

Upstream-Status: Inappropriate [OE specific]
Signed-off-by: Ross Burton <ross.burton@intel.com>

---
 mesonbuild/environment.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mesonbuild/environment.py b/mesonbuild/environment.py
index 267acf9..19a3c1e 100644
--- a/mesonbuild/environment.py
+++ b/mesonbuild/environment.py
@@ -239,7 +239,7 @@ def detect_cpu_family(compilers):
         return 'x86_64'
 
     if trial not in known_cpu_families:
-        raise EnvironmentException('Unknown CPU family %s, please report this at https://github.com/mesonbuild/meson/issues/new' % trial)
+        raise EnvironmentException('Unknown CPU family %s, see https://wiki.yoctoproject.org/wiki/Meson/UnknownCPU for directions.' % trial)
 
     return trial
 
@@ -1012,7 +1012,7 @@ class CrossBuildInfo:
                     raise EnvironmentException('Malformed value in cross file variable %s.' % entry)
 
                 if entry == 'cpu_family' and res not in known_cpu_families:
-                    raise EnvironmentException('Unknown CPU family %s, please report this at https://github.com/mesonbuild/meson/issues/new' % trial)
+                    raise EnvironmentException('Unknown CPU family %s, see https://wiki.yoctoproject.org/wiki/Meson/UnknownCPU for directions.' % value)
 
                 if self.ok_type(res):
                     self.config[s][entry] = res
